src/integrations/watchers/indexable-permalink-watcher.php

```diff
@@ -104,9 +102,29 @@ public function reset_permalinks_term( $old, $new, $type ) {
 	}
 
 	/**
-	 * Retrieves a list with the public post types.
+	 * Resets the permalinks of the indexables.
 	 *
-	 * @codeCoverageIgnore
+	 * @param string      $type    The type of the indexable.
+	 * @param null|string $subtype The subtype. Can be null.
+	 */
+	public function reset_permalink_indexables( $type, $subtype = null ) {
+		$where = [ 'object_type' => $type ];
+
+		if ( $subtype ) {
+			$where['object_sub_type'] = $subtype;
+		}
+
+		Wrapper::get_wpdb()->update(
```

Comment:
Why use `wpdb`? Why not the indexable repository?

Reply:
I have no idea, didn't changed that one.

Reply:
Turns out the method the ORM provides for this does a query for each updated row and is actually completely removed in our own porting of it as we never ever want to do that.

So this can be ignored.